vue.config.js

```diff
@@ -85,14 +83,6 @@ module.exports = {
         ] = `@/env/components/AppNavigation/${envName}.js`;
       }
     }
-
-    if (process.env.NODE_ENV === 'production') {
-      config.plugins.push(
-        new CompressionPlugin({
-          deleteOriginalAssets: true,
-        })
-      );
-    }
```

Comment:
I don't know the web application, so I cannot comment on this technical change.  It seems okay if the web application does not need to consume compressed content.

More important, I do not believe this chnage will fix the problem with the network scanner detecting a possible DEFLATE issue.  The network scanner scans the BMC's HTTPS interface, so if the BMC offers a compressed file -- regardless of if the web application uses it or not -- then the scanner will detect this issue.

Reply:
Resolved in discussion: https://jazz07.rchland.ibm.com:13443/jazz/web/projects/CSSD#action=com.ibm.team.workitem.viewWorkItem&id=579318 - Comment 26